dll/shellext/shellbtrfs/devices.cpp

```diff
@@ -907,7 +907,11 @@ static INT_PTR CALLBACK stub_DeviceResizeDlgProc(HWND hwndDlg, UINT uMsg, WPARAM
 
 void BtrfsDeviceResize::ShowDialog(HWND hwnd, WCHAR* fn, UINT64 dev_id) {
     this->dev_id = dev_id;
+#ifdef __REACTOS__
+    wcscpy_s(this->fn, _countof(fn), fn);
+#else
     wcscpy(this->fn, fn);
```

Comment:
In Microsoft C++, this is not unsafe by the way. It will use this template overload:
```
template <size_t size>  
wchar_t *wcscpy(  
   wchar_t (&strDestination)[size],  
   const wchar_t *strSource   
); // C++ only  
```

Reply:
Therefore we can assure the `wcscpy` there in the code wouldn't cause big problems for us related with buffering? This means I'll just leave it as is and dump the PR?